content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -19,7 +19,7 @@ This document serves both as a reference to the values and as a coordination poi
 
 Example: `app.kubernetes.io/component=database`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
```

Comment:
Remove [caps](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), use [relative links](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects).

```suggestion
Used on: All Objects (typically used on [workload resources](/docs/reference/kubernetes-api/workload-resources/)).
```